src/components/card/card.js

```diff
@@ -8,7 +8,8 @@
 angular.module('material.components.card', [
     'material.core'
   ])
-  .directive('mdCard', mdCardDirective);
+  .directive('mdCard', mdCardDirective)
+  .directive('mdPlainImage', mdPlainImageDirective);
```

Comment:
@crisbeto, @topherfangio - we already have 

```html
<md-card-header>
  <md-card-avatar>
    <img class="md-user-avatar" src="avatar.png"/>
  </md-card-avatar>
</md-card-header> 
```
And we also have a `.md-card-image` class:

```html
<md-card>
 <img ng-src="card-image.png" class="md-card-image" alt="image caption">
</md-card>
```

So I think we should not use .md-plain-image and instead improve the class:

```css
.md-card-image {
// We need to use !important, because md-card also uses it.
height: auto !important;
width: auto;
max-width: 100%;
}
```